analysis_driver/quality_control/relatedness.py

```diff
@@ -154,10 +154,14 @@ class GenotypeGVCFs(RelatednessStage):
     gVCFs = ListParameter()
     reference = Parameter()
 
+    @property
+    def memory(self):
+        return max(len(self.gVCFs) * 3, 50)
+
     def gatk_genotype_gvcfs_cmd(self):
```

Comment:
I would prefer if `number_threads` and `memory` were parameters of `gatk_genotype_gvcfs_cmd` and passed from `_run` function